Automattic/newspack-plugin
# feat: campaign group management ui

### All Submissions:






### Changes proposed in this Pull Request:

Adds a "Groups" tab for managing Campaign Groups with basic functionality. Groups can be created and deleted here. Some ideas for future iterations on this screen:

- Ability to rename groups
- Ability to Publish/Unpublish all campaigns in the group (redundant to Campaigns tab?)
- Ability to Preview

### How to test the changes in this Pull Request:

1. Observe new Groups tab in Campaigns wizard.
2. Try creating and deleting groups here.

### Other information:
